lib/money_tracker.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'transactions.rb'
+require 'date'
+
+class MoneyTracker
+
+  attr_reader :balance, :transactions
+
+  def initialize
+    @balance = 0
+    @transactions = Transactions.new
+  end
+
+  def credit(amount)
+    @balance += amount
```

Comment:
Is it possible to only calculate balance when its needed for printing the statement?